apps/learninglocker/templates/nginx/_configs/default.conf.j2

```diff
@@ -0,0 +1,72 @@
+upstream backend_ui {
+  server learninglocker-ui:{{ learninglocker_ui_port }};
+}
+
+upstream backend_xapi {
+  server learninglocker-xapi:{{ learninglocker_xapi_port }};
+}
+
+upstream backend_api {
+  server learninglocker-api:{{ learninglocker_api_port }};
+}
+
+server {
+  listen       {{ learninglocker_nginx_port }} default_server;
+  listen       [::]:{{ learninglocker_nginx_port }} default_server;
```

Comment:
Can you add a comment to explain this please? I don't get it.

Reply:
It's for binding IPV6 addresses

Reply:
Okay then. Add a comment please.